src/univalue/configure.ac

```diff
@@ -52,8 +52,19 @@ AC_ARG_ENABLE([debug],
     [enable_debug=$enableval],
     [enable_debug=no])
 
```

Comment:
why adding this to `univalue` and not to other subsytems, e.g `libsecp`?

Reply:
univalue is the only sub-component that requires these flags if bitcoind has set them because it uses boost libraries (program_options).  The problem is that program_options uses std::set, which is going to actually be the symbol std::set or std::__debug::set based on these flags.  So if in univalue you use std::set and in bitcoind you use std::__debug::set, then the boost library doesn't provide the proper symbols for one or the other.